cms/djangoapps/contentstore/views/block.py

```diff
@@ -135,6 +135,8 @@ def xblock_handler(request, usage_key_string=None):
                      if duplicate_source_locator is not present
                 :staged_content: use "clipboard" to paste from the OLX user's clipboard. (Incompatible with all other
                      fields except parent_locator)
+                :library_content_key: the key of the library content to add. (Incompatible with
+                     all other fields except parent_locator)
```

Comment:
Having this POST arg will be nice, so that we don't have to follow up the request with a PUT 👍🏻

Reply:
@kdmccormick Yes, called upstream sync after creating the block in a single call, simplified the js part. https://github.com/openedx/edx-platform/pull/35670/commits/bc27ba347a56a428755038d7289aa269429ff879